execute.py

```
@@ -1,127 +1,245 @@
 import pandas as pd
 import numpy as np
+import matplotlib.pyplot as plt
```

Ruff (0.8.0)

`matplotlib.pyplot` imported but unused

Remove unused import: `matplotlib.pyplot`

(F401)